releasenotes/README.md

```diff
@@ -155,83 +31,87 @@ Run `sfdx plugins --core` to display the version of Salesforce CLI and the insta
+### Removed Commands
 
-* `force:package:version:delete`
-* `force:package:delete`
+These commands were removed in v51. The Hammer Test closed pilot program has been terminated.
 
-    Deletes the package or package version.
+* `force:package:hammertest:list`
+* `force:package:hammertest:report`
+* `force:package:hammertest:run`
 
-    You can't recover deleted packages or package versions, or specify them as dependencies. New installations of the package will fail. You can’t delete released versions of second-generation managed packages.
+### New Parameters
 
-### Removed Commands
+* `force:mdapi:deploy --soapdeploy`
 
-These commands were deprecated in v49 and removed in v50.  You can continue using them by installing the [Lightning Testing Service CLI Plug-in](https://github.com/forcedotcom/plugin-lightning-test-service). Salesforce doesn't support or update this plug-in.
+  The `soapdeploy` parameter will cause the metadata deploy to use the SOAP API, instead of the REST API.
 
-* `force:lightning:test:install`
-* `force:lightning:test:run`
+### Deprecated Parameters
 
-### New Parameters
+* `force:org:shape:create`
 
-* `force:package:create --errornotificationusername (-o)`
-* `force:package:update --errornotificationusername (-o)`
+  The `definitionfile` parameter has been removed from this command. This parameter is part of the Org Shape feature which is currently in beta and has been removed based on feedback that users prefer storing Org Shape in the shape source org instead of a scratch definition file.
 
-    Designate an active Dev Hub org user to receive email notifications for unhandled Apex exceptions, as well as install, upgrade, or uninstall failures associated with your packages. Specify the username during package creation, or when you update the package with the `force:package:update` command.
+### Second Generation Packaging
 
-### Other Changes
+* The release version stamped on a package version is now available in the response of command `force:package:version:report` when the user specifies the `--verbose` parameter.
 
-* When parsing the `.forceignore` file, Salesforce CLI uses the same rules and patterns as [git uses with the `.gitignore` file](https://git-scm.com/docs/gitignore). As a result, some entries in your existing `.forceignore` files might be invalid. The `force:source:*` commands warn you about any invalid entries.  
+* Package version CLI commands are enhanced to support component deletion
+  * If force:package:version:create completes successfully and if any metadata has been removed from the package version in comparison to its ancestor, a generic warning message is shown in the `force:package:version:create` output highlighting that some metadata was removed.
+  * Command `force:package:version:list` displays this info (boolean, whether or not metadata was removed) when used with --verbose param
```

Comment:
This is confusing.  Not sure where the "boolean" fits in.  Can we just say  "The command `force:package:version:list --verbose` displays whether metadata was removed."?